spec/poll_spec.rb

```diff
@@ -1,13 +1,15 @@
 require "spec_helper"
+require "bbbevents"
 
 RSpec.describe BBBEvents::Poll do
   before(:all) do
-    @poll = @sample.polls.first
+    @sample1 = BBBEvents.parse(file_fixture("new-events-2_6.xml"))
```

Comment:
You've switched all the tests to run on the 2.6 poll data, but now they're no longer running on the old poll data. Instead of changing the existing tests, some new tests for the new events should be added.

Reply:
Now uses new-events-2_5.xml.